File: Code_Python/Exercicio-09/peça-uma-nota-entre-0-100.py
#atv em aula
nota = int(input("Digite uma nota entre Zero e Cem: "))
while (nota < 0) or (nota > 100):
        print("[Alerta: Número inválido]")
        nota = int(input("Digite um Valor válido: "))
        break
print("Esse numero é válido: ", nota)

File: Code_C/Exercicio-07/peça-uma-nota-entre-0-100.c
#include <stdio.h>
#include <stdlib.h>

void pausar(){
	printf("\nPressione alguma tecla para continuar...");
	getch();
}

int main(){
	int nota;
	
	printf("Digite um numero entre Zero e Cem: \n");
	scanf("%d", &nota);
	
	while ((nota < 0) || (nota > 100)){
		printf("[Alerta: Esse numero [%d] e invalido.] \n", nota);
		printf("Digite um numero entre Zero e Cem: \n");
		scanf("%d", &nota);
	}
	printf("[Esse numero [%d] esta entre Zero e Cem.] \n", nota);
    
 	pausar();
 	return(0);
}
